src/components/Enemy.ts

```diff
@@ -96,6 +98,7 @@ export default class Enemy extends Phaser.Sprite {
 
     public kill(): any {
         super.kill();
+        this.killAudio.play();
```

Comment:
maybe it's better to play the sound on hit function, as kill() is called to destroy the instances on starting a second game. or we make a separate issue to stop using kill() in game create (https://github.com/DankGamesStudios/Tankr/blob/master/src/states/game.ts#L92)

Reply:
Done